symphony-bdk-examples/bdk-app-spring-boot-example/src/main/resources/application.yaml

```diff
@@ -3,18 +3,32 @@ bdk:
   host: acme.symphony.com
   bot:
     username: mybot
-    privateKeyPath: /Users/user.name/.symphony/privatekey.pem
+    privateKeyPath: ${user.home}/.symphony/privatekey.pem
+
   app:
     appId: myapp
-    privateKeyPath: /Users/user.name/.symphony/privatekey.pem
+    privateKeyPath: ${user.home}/.symphony/privatekey.pem
 
 bdk-app:
   auth:
     enabled: true
     jwtCookie:
       enabled: true
       maxAge: 1d
+  cors:
+    "[/**]":
+      allowed-origins: "*"
```

Comment:
Put that [here](https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blob/master/docs/spring-boot/app-starter.md#create-a-simple-backend-for-extension-application) as well: doc, doc and doc 😉